src/index.css

```
@@ -0,0 +1,43 @@
+@import url('https://fonts.googleapis.com/css2?family=Alegreya+Sans:wght@400;500;700&family=Zen+Kaku+Gothic+New:wght@400;500;700&display=swap');
```

Stylelint (17.3.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Alegreya+Sans:wght@400;500;700&family=Zen+Kaku+Gothic+New:wght@400;500;700&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Alegreya+Sans:wght@400;500;700&family=Zen+Kaku+Gothic+New:wght@400;500;700&display=swap'" (import-notation)

(import-notation)